tests/test_jx/test_metadata.py

```diff
@@ -120,29 +120,28 @@ def test_get_nested_columns(self):
         test = {
             "query": {
                 "select": ["name", "table", "type", "nested_path"],
-                "from": "meta.columns",
-                "where": {"term": {"table": table_name}}
+                "from": "meta.columns"
             },
             "expecting_list": {
                 "meta": {"format": "list"},
                 "data": [
-                    {"table": table_name, "name": "_id", "type": "string", "nested_path": "."},
-                    {"table": table_name, "name": "_a", "type": "nested", "nested_path": "."},
-                    {"table": table_name, "name": "_a.b", "type": "string", "nested_path": ["_a", "."]},
-                    {"table": table_name, "name": "_a.v", "type": "double", "nested_path": ["_a", "."]},
-                    {"table": table_name, "name": "c", "type": "string", "nested_path": "."},
-                    {"table": table_name, "name": "o", "type": "double", "nested_path": "."},
+                    {"table": table_name, "name": "_id", "type": "string", "nested_path": ["."]},
+                    {"table": table_name, "name": "_a", "type": "nested", "nested_path": ["."]},
+                    {"table": table_name + "._a", "name": "_a.b", "type": "string", "nested_path": ["_a", "."]},
+                    {"table": table_name + "._a", "name": "_a.v", "type": "number", "nested_path": ["_a", "."]},
+                    {"table": table_name, "name": "c", "type": "string", "nested_path": ["."]},
+                    {"table": table_name, "name": "o", "type": "number", "nested_path": ["."]},
                 ]},
             "expecting_table": {
                 "meta": {"format": "table"},
                 "header": ["table", "name", "nested_path", "type"],
                 "data": [
-                    [table_name, "_id", ".", "string"],
-                    [table_name, "_a", ".", "nested"],
-                    [table_name, "_a.b", ["_a", "."], "string"],
-                    [table_name, "_a.v", ["_a", "."], "double"],
-                    [table_name, "c", ".", "string"],
-                    [table_name, "o", ".", "double"]
+                    [table_name, "_id", ["."], "string"],
```

Comment:
the nested path should be just dot (`"."`), not an array with a dot (`["."]`).  JSON query expressions make no distinction between these two forms, and chooses the simplest when possible.  Use `unwraplist()` to do this for you.